docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,308 @@
+| DEM source                                                | DEM used in processing                                                                                      | GLO-30                                                                                                      |
+| DEM resolution                                            | Pixel spacing in meters for DEM used to process this scene                                                  | 30                                                                                                          |
+| Unwrapping type                                           | Phase unwrapping algorithm used                                                                             | snaphu_mcf                                                                                                  |
+| Speckle filter                                            | Speckle filter applied                                                                                      | yes                                                                                                         |
+
+*Table 4: List of InSAR parameters included in the parameter text file*
+
+{% endblock %}
+
+{% block line_of_sight %}
+### Line-of-Sight Measurements
+When looking at a single interferogram, the deformation measurements in the line-of-sight orientation of the sensor indicate relative motion towards or away from the sensor. InSAR is not sensitive to motion in the azimuth direction of the satellite, so motion that occurs in the same direction as the satellite's direction of travel will not be detected.
+
+A single interferogram cannot be used to determine the relative contributions of vertical and horizontal movement to the line-of-sight displacement measurement. To determine how much of the signal is driven by vertical vs. horizontal movement, you must either use a time series of interferograms, or use reference measurements with known vertical and horizontal components (such as GNSS measurements from the region of deformation) to deconstruct the line-of-sight displacement.
+
+{% endblock %}
+
+{% block reference_point %}
+### Phase Unwrapping Reference Point
+The reference point for phase unwrapping is set automatically by the topsApp.py script. It may not be an ideal location to use as a reference point for phase unwrapping. If it is located in an area undergoing deformation, or in an area with low coherence, the unwrapping may be of lower quality than if the reference point was in a more suitable location.
+
+Even when there are not phase unwrapping errors introduced by phase discontinuities, it is important to be aware that unwrapped phase differences are calculated relative to the reference point.[(TODO: check if this is the same with ISCE2:) The phase difference value of the reference point is set to 0 during phase unwrapping, so any displacement values will be relative to that benchmark.] 
```

Comment:
Verify what we know (or don't) about how topsApp selects a reference point. Maybe @forrestfwilliams has some insight?

Reply:
This section looks accurate to me. I don't have any more information on how topsApp picks the reference, but I don't think we need to add any more information.

Reply:
OK. I've cleaned up the TODO, so these comments should be reflected in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/360/commits/122437f86a3c7268fd037e07e7c964408797ed4a)